R/htbatch.R

```diff
@@ -23,31 +23,38 @@ htbatch <- function (file, sigma.selection, ...) {
       rm (t)
 
 
+      if (is.na(qsardataframe$r1.ind.smiles[i]) == FALSE) {
 
-    } else {
+        t <- htdesc (smile = qsardataframe$r1.ind.smiles[i], HT.type = "induction", sigma.selection)
+        qsardataframe$r1.ind.sub.smiles[i] <- as.character (t$sub)
+        qsardataframe$r1.ind.mcs.index[i] <- t$tanimoto
+        qsardataframe$r1.ind.value[i] <- t$value
+        rm (t)
 
-      # if its not aromatic, than we will need to call htdesc
+      }
 
-      #converting SMIset into SDFSet required for evaluation in getfragmentsigma
 
-      # getting sigma value from htdescvalue function for r1taft
 
-      # calling htdesc to fill substitute mcs values
+    } else {
 
       # if the structure is aromatic, than we will set these to default values
 
-      # qsardataframe$r1.taft.smiles[i] <- "*C1=CC=CC=C1"
-      # qsardataframe$r1.es.smiles[i] <- "*C1=CC=CC=C1"
-      # qsardataframe$r1.ind.smiles[i] <- "*C1=CC=CC=C1"
+      qsardataframe$r1.taft.smiles[i] <- "*C1=CC=CC=C1"
+      qsardataframe$r1.ind.smiles[i] <- "*C1=CC=CC=C1"
```

Comment:
If the structure is indeed aromatic, than we will the same default SMILES for es as we do for taft and ind; so add that line back (Line 40 deleted one).

Reply:
Ok, what should we do about steric fragments that exist as substituted aromatics? Won't this have them only represented as phenyl?

Reply:
Thanks for the clarification John. I didnt know that we had substituted phenyls for Es in the library. In that case, we will have the leave the logic as is and make the htdesc call.

Reply:
@michaela thanks for creating a helper func. I will have to manually resolve merge conflicts so will do it later today.

Reply:
Actually I'm wrong. It appears we don't have fragments for substituted phenyls in the Es library.  There are some in other libraries but now that I checked not Es. Sorry about that

Reply:
Oh ok so what do you guys propose now? I can close this PR for now (without merging) and just push a new PR after discussing it with michaela and others at EPA.